src/app/pages/RoflAppDetailsPage/index.tsx

```diff
@@ -96,124 +105,224 @@ export const StyledGrid = styled(Grid)(({ theme }) => ({
-              {app.id}
-            </Typography>
-            <CopyToClipboard value={app.id} />
-          </dd>
-
-          <dt>{t('rofl.enclaveId')}</dt>
-          <dd>
-            <Enclaves policy={app.policy} />
-          </dd>
-
-          <dt>{t('rofl.sekPublicKey')}</dt>
-          <dd>
-            <Typography variant="mono">{app.sek}</Typography> <CopyToClipboard value={app.sek} />
-          </dd>
-
-          <dt>{t('rofl.adminAccount')}</dt>
-          <dd>
-            <AccountLink
-              scope={{ network: app.network, layer: app.layer }}
-              address={app.admin_eth ?? app.admin}
-              alwaysTrimOnTablet
-            />
-            <CopyToClipboard value={app.admin_eth ?? app.admin} />
-          </dd>
-
-          <dt>{t('rofl.stakedAmount')}</dt>
-          <dd>
-            {app.stake
-              ? t('common.valueInToken', {
-                  ...getPreciseNumberFormat(app.stake),
-                  ticker: app.ticker,
-                })
-              : t('common.missing')}
-          </dd>
-        </>
-      )}
-
-      <dt>{t('common.status')}</dt>
-      <dd>
-        <RoflAppStatusBadge hasActiveInstances={!!app.num_active_instances} removed={app.removed} />
-      </dd>
-
-      {!detailsPage && (
-        <>
-          <dt>{t('rofl.appId')}</dt>
-          <dd>
-            <RoflAppLink id={app.id} network={app.network} withSourceIndicator={false} />
-          </dd>
-          <dt>{t('rofl.instances')}</dt>
-          <dd>{app.num_active_instances.toLocaleString()}</dd>
-          <dt>{t('rofl.created')}</dt>
-          <dd>
-            {t('common.formattedDateTime', {
-              value: app.date_created,
-              formatParams: {
-                timestamp: {
-                  year: 'numeric',
-                  month: 'long',
-                  day: 'numeric',
-                } satisfies Intl.DateTimeFormatOptions,
-              },
-            })}
-          </dd>
-          <dt>{t('rofl.lastActivity')}</dt>
-          <dd>{<TableCellAge sinceTimestamp={app.last_activity} />}</dd>
-        </>
-      )}
-
-      {detailsPage && (
-        <>
-          <dt>{t('rofl.lastActivity')}</dt>
-          <dd>
-            <LastActivity
-              scope={{ network: app.network, layer: app.layer }}
-              transaction={app.last_activity_tx}
-            />
-          </dd>
-
-          <dt>{t('rofl.activeInstances')}</dt>
-          <dd>{app.num_active_instances.toLocaleString()}</dd>
-
-          <dt>{t('rofl.endorsement')}</dt>
-          <dd>
-            <Endorsement policy={app.policy} />
-          </dd>
-
-          <dt>{t('rofl.secrets')}</dt>
-          <dd>
-            <Secrets secrets={app.secrets} />
-          </dd>
-        </>
-      )}
+    <StyledDescriptionList titleWidth={isMobile ? '100px' : '200px'}>
+      <NameRow name={app.metadata['net.oasis.rofl.name']} />
+      <VersionRow version={app.metadata['net.oasis.rofl.version']} />
```

Comment:
Metadata props are confusing 😅

Reply:
indeed, but there is nothing I can do https://github.com/oasisprotocol/explorer/blob/master/src/app/pages/RoflAppDetailsPage/MetaDataCard.tsx#L29